### PR TITLE
replaced taxiinvaranasi with kashitax.in

### PR DESCRIPTION
**Pull request by AdarshKumar02291**

### CODE REVIEW

**Comment by AdarshKumar02291** [comment]
update keyword and image

File: app/src/components/about/About.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function About() {
                                 <span><BsClockFill className="font-20 text-orange" /> Mon-Sat 8:00-18:00 </span>
                             </div>
                             <div className="font-regular font-15 text-brown mb-2">
-                                <span><GrMail className="font-20 text-orange" /> taxiinvaranasiii@gmail.com</span>
+                                <span><GrMail className="font-20 text-orange" /> kashitaxi@gmail.com</span>
```
Comment (DEVAbhinav) [inline app/src/components/about/About.jsx:35]: Email change not required. @dev-1ce

File: app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Kashi-Vishwanath-Darshan",
-    "image": "../images/blogKashivishwanath.png",
+    "image": "../images/blogKashiVishwanath.jpg",
```
Comment (DEVAbhinav) [inline app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json:3]: Why this change?

Comment (AdarshKumar02291) [reply app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json:3]: replaced old kashiVishwanath image with new chat gpt one . renamed it same as old one just the format was different

File: app/src/components/about/About.jsx (hunk 1 above)
Comment (AdarshKumar02291) [reply app/src/components/about/About.jsx:35]: will fix this now and sending PR

File: app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json (hunk 2 above)
Comment (DEVAbhinav) [reply app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json:3]: Okay

Comment (DEVAbhinav) [reply app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json:3]: Please add Screenshot

File: app/src/components/about/About.jsx (hunk 1 above)
Comment (AdarshKumar02291) [reply app/src/components/about/About.jsx:35]: Merge it for now will fix it in next PR
